demos/simple_discrete_uniform_distribution.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import cm
+
```

Comment:
`from mpl_toolkits.mplot3d import Axes3D`

needed to run the 3d projection for me

Reply:
same for the other demos